frontend/src/App.js

```diff
@@ -1,24 +1,42 @@
 import React, { Fragment } from 'react';
 import './App.css';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
-import { Products, SingleProduct, WishList, Landing, Cart } from './screens';
-import { Navbar } from './components'
+import { Products, SingleProduct, WishList, Landing, Cart, Register, Login } from './screens';
+import { Navbar } from './components';
+import PrivateRoute from './routes/Privateroute';
+import 'react-toastify/dist/ReactToastify.css';
+import { ToastContainer } from 'react-toastify';
+
 
 const App = () => {
+
   return (
     <div className="app">
       <Router>
         <Switch>
           <Fragment>
```

Comment:
What is the benefit of using a Fragment here?
If it is just used to wrap all the routes, they are being done within `<Switch>` right?

Reply:
Using of Fragment here is, Since I have a navbar just below that, which is not a route. So to use LINK tag inside that(Navbar)
It is mandatory to wrap into fragment or some kind of div, I suppose.
Otherwise using LINK tag inside Navbar would result into error
Thanks